.devcontainer/build.sh

```diff
@@ -14,5 +14,5 @@ yarn workspace @affine/server-native build
 # Create database
 yarn workspace @affine/server prisma db push
 
-# Create user username: affine, password: affine
+# Create user username: affine, email: affine@affine.pro, password: affine
```

Comment:
this script is outdated

Reply:
so I should just delete this line and update the comments?

Reply:
> so I should just delete this line and update the comments?

recommand add comments to doc